src/drivers/driver_zephyr.c

```diff
@@ -1336,7 +1336,6 @@ static int wpa_drv_zep_associate(void *priv,
 		goto out;
```

Comment:
Typo in commit subject, `Rremove` should be `Remove`

Also it would be nice to see description why this change is needed in the commit message.

Reply:
updated, pls review again